src/components/Layout/Header.tsx

```diff
@@ -73,10 +73,16 @@ const Header: FunctionComponent<Props> = props => {
                                 <NavDropdown.Item href="/podcast/">Sourcegraph Podcast</NavDropdown.Item>
                             </NavDropdown>
 
-                            <NavDropdown id="customersDropdown" title="Customers">
-                                <NavDropdown.Item href="/case-studies">Case studies</NavDropdown.Item>
-                                <NavDropdown.Item href="/use-cases">Use cases</NavDropdown.Item>
-                                <NavDropdown.Item href="/contact/product-specialist">Become one</NavDropdown.Item>
+                            <NavDropdown id="useCaseDropdown" title="Use Cases">
```

Comment:
Opening a new tab on desktop makes sense to me, and I agree that those links should behave the same. Thank you for catching this! 

Similarly, it seems we should clean up inconsistencies in the external-facing mobile links as well. What do you think of keeping these links in the "same tab" on mobile? I was reading [this article](https://uxdesign.cc/linking-to-a-new-tab-vs-same-tab-f88b495d2187#ba83) for reference, but it is just one perspective. I am unsure what is preferred and open to the standard we decide on together.

Reply:
I had a discussion with Brett on the opening new tabs issue [here](https://github.com/sourcegraph/about-replatform/pull/123#discussion_r861373447). In my opinion, if the link is coming from the Nav Bar, as a user, I'd expect it's for the same site, so I would be thrown off to enter a new site. I think in this case it's a courtesy to keep their spot on /about. Other links outside of the Nav Bar should open in the same tab, though. This is just my opinion though I know it's a heated topic hah. Curious what @bretthayes  thinks.

Either way, whatever we land on, the Nav Bar links to other sites should have the same behavior, open in same tab or new tab.

Reply:
This context was very helpful. Thank you! From what I gathered, we prefer `_blank` _with_ `noreferrer` due to linting. Please let me know if I misinterpreted. 

As you requested, I made all of the outgoing links in the nav bar open in new tabs on both desktop and mobile: fbee388. The only exception I made was for Search Code and Sign In to make the product quicker to access. Because About is the main site, we may expect some users navigating to Search this way instead of typing `/search`. In this case, they would already anticipate the destination and probably wouldn't want to navigate to a new tab by default.

Pending @bretthayes' input as well! Curious to hear what you think of desktop vs. mobile.

Reply:
> Branding branding branding!

I cannot and will not get over this ^